scholarx/2021/styles.css

```diff
@@ -119,3 +119,7 @@
 {
     color: #fbc53b;
 }
+
+#past-onelives{
+    background-color: white;
+}
```

Comment:
Shall we use `bg-white` (I can't remember the name, you'll have to double-check this), class in bootstrap theme instead?

Reply:
I added the class name as suggested, kindly check if that is what you meant. Thanks